### PR TITLE
Adding tenant Id property to teamworkuseridentity.md

### PR DESCRIPTION
**Pull request by evanypark**
Introducing new property, tenantId to teamworkUserIdentity.md
Updating entity type of user from identity.md to teamworkUserIdentity.md

### CODE REVIEW

File: api-reference/beta/resources/chatmessagefromidentityset.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Inherits from [identitySet](../resources/identityset.md).
 |:---|:---|:---|
 |application|[identity](../resources/identity.md)|Inherited from [identitySet](../resources/identityset.md). If present, represents the application (for example, bot) that sent the message.|
 |device|[identity](../resources/identity.md)|Inherited from [identitySet](../resources/identityset.md). Not used.|
-|user|[identity](../resources/identity.md)|Inherited from [identitySet](../resources/identityset.md). If present, represents the user that sent the message.|
+|user|[teamworkUserIdentity](../resources/teamworkUserIdentity.md)|If present, represents the user that sent the message.|
```
Comment (Lauragra) [inline api-reference/beta/resources/chatmessagefromidentityset.md:26]: This change warrants a changelog entry.

File: api-reference/beta/resources/teamworkuseridentity.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Inherits from [identity](../resources/identity.md).
 |displayName|String|Inherited from [identity](../resources/identity.md). Display name of the user. Optional.|
 |id|String|Inherited from [identity](../resources/identity.md). ID of the user. |
 |userIdentityType|teamworkUserIdentityType| Type of user. Possible values are: `aadUser`, `onPremiseAadUser`, `anonymousGuest`, `federatedUser`, `personalMicrosoftAccountUser`, `skypeUser`, `phoneUser`, `emailUser` and `azureCommunicationServicesUser`.|
+|tenantId|String|Identifier of tenant, which user is part of. Optional. |
```
Comment (Lauragra) [inline api-reference/beta/resources/teamworkuseridentity.md:27]: You'll want a changelog entry for this new property too. (And for v1.0.)

File: api-reference/beta/resources/chatmessagefromidentityset.md (hunk 1 above)
Comment (evanypark) [reply api-reference/beta/resources/chatmessagefromidentityset.md:26]: Change log entry has been added for this change

File: api-reference/beta/resources/teamworkuseridentity.md (hunk 2 above)
Comment (evanypark) [reply api-reference/beta/resources/teamworkuseridentity.md:27]: Change log entry has been added for this change

**Review by Lauragra — APPROVED** [review]

**Pull request merged**